package.json

```diff
@@ -80,6 +80,7 @@
     "popper.js": "^1.12.6"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-object-rest-spread": "^7.0.0-beta.2",
```

Comment:
Uhh you should make sure you are using the same version plugin and without the `^`?

`"babel-plugin-transform-object-rest-spread": "7.0.0-beta.2",` (or just update everything to scoped)

Reply:
@hzoo: can you please make a PR against v4-dev to update all babel packages to the latest versions and CC me? I tried with scoped packages and 7.0.0-beta.5 but then I got some error in ESLint.

I'd like to pinpoint any potential issues and report them upstream so that they get fixed.

Thanks!

Reply:
@Johann-S: for this branch just remove `^` like @hzoo says above and like we do with the other babel plugins already.

Reply:
done 👍